Assets/GothicVR/Scripts/Data/SkyState.cs

```diff
@@ -18,6 +18,8 @@ public class SkyState
         public int cloudShadowOn;
         public SkyLayerData[] layer;
 
+        public float lerpDuration = 0.05f;
```

Comment:
Please add a comment why this value (from original G1?) and what it changes if altered.